docs/docs/kds-rest-api.md

```diff
@@ -115,12 +117,12 @@ cannot be accessed by the altered key specification.
 }
 ```
 
-| Field | Description |
-| ----- | ----------- |
-| appAttestationReport | The `EnclaveInstanceInfo` in Base64 of the application enclave requesting access to the private key. This will be validated against the policy referred to by the key specification. The private key will be returned only if the policy check passes. |
-| name | A name for the key. The name is used during key derivation so can be used to ensure that keys with the same master key and constraint configuration can be uniquely generated. |
-| masterKeyType | The type of master key provider that is used to source the master key. |
-| policyConstraint | The constraint policy to apply to the key. A key will only be released to an enclave that meets these constraints. |
+| Field | Description                                                                                                                                                                           |
+| ----- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| appAttestationReport | The `EnclaveInstanceInfo` in Base64 of the application enclave requesting access to the private key. The KDS validates this field against the policy referred in the key specification. The KDS returns the private key only if the policy check passes. |
+| name | A name for the key. The KDS uses the name field to generate unique keys with the same master key and constraint configuration. |
+| masterKeyType | The type of master key provider the KDS should use to source the master key.                                          |
+| policyConstraint | The constraint policy to apply to the key. The KDS will reveal the keys only to an enclave that meets these constraints. |
```

Comment:
Link to policy constraint DSL documentation.

Reply:
Shall I link to this section?
https://r3-conclave.artifacts.teamcity.com/presignedTokenAuth/b097f4ec-7f6b-44e3-9f95-cfb551979a9c/repository/download/ConclaveCore_SdkDocs/4910:id/docs.zip!/kds-configuration.html#policy-constraint-section-policyconstraint

Reply:
Not quite, this is what you should link to: https://r3-conclave.artifacts.teamcity.com/presignedTokenAuth/3f891a7a-59be-4d4b-b55e-f054d92e5961/repository/download/ConclaveCore_SdkDocs/4910:id/docs.zip!/constraints.html